crates/bevy_pbr/src/light/mod.rs

```diff
@@ -522,6 +523,26 @@ pub enum SimulationLightSystems {
     CheckLightVisibility,
 }
 
+pub fn map_ambient_lights(
```

Comment:
I think we may want to add a deprecation notice to the AmbientLight component and remove it after 0.16. For this PR I think this is fine as is though.

Reply:
+1 to adding a deprecated attribute to ambient light, and then removing this system in 0.17.